src/Components/pages/Home.jsx

```diff
@@ -1,45 +1,75 @@
 import React, {Component,Fragment} from 'react';
 import {Helmet} from 'react-helmet';
-import {Link, withRouter} from 'react-router-dom';
-
-import UserProvider from "../authenticate/providers/UserProvider";
-import { UserContext } from "../authenticate/providers/UserProvider";
-
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import {Link} from 'react-router-dom';
 
 class Home extends Component {
+    
     constructor(props) {
         super(props);
-        this.state={};
+        this.state ={user : 'N/A'};
+    }
+    
+    componentDidMount() 
+    {
+        let userAuth = null;
+        if( localStorage.getItem('User') == 'N/A')
+            userAuth = 'N/A'
+        else
+            userAuth = localStorage.getItem('User')
+        window.addEventListener("storage", e =>{
+            let change = localStorage.getItem('User')
+            this.setState({ user: change});
+        });
+        this.setState({user:userAuth})
+        const {gameId} = this.state;
+    }
```

Comment:
is this the same code as the componentDidMount from the create page?? if so can you explain why it's the same/try to find a way to reuse the code without rewriting it?